src/cc65/compile.c

```diff
@@ -300,7 +299,7 @@ void Compile (const char* FileName)
     struct tm*  TM;
 
     /* Since strftime is locale dependent, we need the abbreviated month names
-    ** in english.
+    ** in English.
```

Comment:
Do you mean that the [Cambridge Dictionary](http://dictionary.cambridge.org/dictionary/english/english?q=English) and [Wikipedia](https://en.wikipedia.org/wiki/English_language) are wrong?